src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx

```diff
@@ -16,30 +16,138 @@
 
 import React from 'react';
 import { useSelector } from 'react-redux';
+import { gql, useQuery } from '@apollo/client';
 
-import { getDisplayStableId } from 'src/shared/state/ens-object/ensObjectHelpers';
+import * as urlFor from 'src/shared/helpers/urlHelper';
+import { getFormattedLocation } from 'src/shared/helpers/formatters/regionFormatter';
+import { getStrandDisplayName } from 'src/shared/helpers/formatters/strandFormatter';
+import {
+  buildFocusIdForUrl,
+  getDisplayStableId
+} from 'src/shared/state/ens-object/ensObjectHelpers';
 import { getBrowserActiveEnsObject } from 'src/content/app/browser/browserSelectors';
 
+import ViewInApp from 'src/shared/components/view-in-app/ViewInApp';
+
 import { EnsObjectGene } from 'src/shared/state/ens-object/ensObjectTypes';
+import { Gene as GeneFromGraphql } from 'src/shared/types/thoas/gene';
+
+import styles from './GeneSummary.scss';
+
+const GENE_QUERY = gql`
+  query Gene($genomeId: String!, $geneId: String!) {
+    gene(byId: { genome_id: $genomeId, stable_id: $geneId }) {
+      alternative_symbols
+      name
+      stable_id
+      symbol
+      so_term
+      transcripts {
+        stable_id
+      }
+      slice {
+        strand {
+          code
+          value
+        }
+        location {
+          length
+        }
+      }
+    }
+  }
+`;
 
-import styles from 'src/content/app/browser/drawer/Drawer.scss';
+type Gene = Required<
+  Pick<
+    GeneFromGraphql,
+    | 'stable_id'
+    | 'symbol'
+    | 'name'
+    | 'alternative_symbols'
+    | 'so_term'
+    | 'transcripts'
+    | 'slice'
+  >
+>;
 
 const GeneSummary = () => {
-  const ensObject = useSelector(getBrowserActiveEnsObject) as EnsObjectGene;
+  const ensObjectGene = useSelector(getBrowserActiveEnsObject) as EnsObjectGene;
+
+  const { data, loading } = useQuery<{ gene: Gene }>(GENE_QUERY, {
+    variables: {
+      geneId: ensObjectGene.stable_id,
+      genomeId: ensObjectGene.genome_id
+    },
+    skip: !ensObjectGene.stable_id
+  });
+
+  if (loading || !data?.gene) {
+    return null;
+  }
+
+  if (!data?.gene) {
+    return <div>No data available</div>;
+  }
+
+  const { gene } = data;
+
+  const stableId = getDisplayStableId(gene);
+
+  const focusId = buildFocusIdForUrl({
+    type: 'gene',
+    objectId: gene.stable_id as string
+  });
+  const entityViewerUrl = urlFor.entityViewer({
+    genomeId: ensObjectGene.genome_id,
+    entityId: focusId
+  });
 
   return (
-    <div className={styles.drawerView}>
-      <div className={styles.container}>
+    <div className={styles.container}>
+      <div className={styles.standardLabelValue}>
```

Comment:
What does standardLabelValue mean?

![image](https://user-images.githubusercontent.com/6834224/107382097-6cc87c00-6ae7-11eb-9232-b9d25b4ba982.png)

Reply:
It's just a name I've given to the container that has the label field on the left and the value on the right with 10px gap in between. I have renamed it to `row` for now.